src/core/schema/config.model.schema

```diff
@@ -291,6 +291,19 @@
     },
     "themeSettings": {
       "type": "object"
+    },
+    "_inview": {
+      "type": "object",
+      "title": "jquery.onscreen (inview)",
+      "properties": {
+      "_allowScrollOver": {
+        "type": "boolean",
+        "default": true,
+        "title": "Allow scroll over",
+        "inputType": "Checkbox",
+        "help": "Some browsers pause script execution during scroll (e.g. some versions of iOS). If this option is selected the inview library will determine if elements became visible during the scroll event."
```

Comment:
I think this help text needs re-wording a bit so it is more helpful to the (generally-speaking) less technical authoring tool user. Probably need to resolve the issue of the correct default setting before we can improve on this.

Reply:
"Some browsers pause script execution during scroll (e.g. some versions of iOS). If this option is selected the inview library will determine if elements became visible during the scroll event."

This would be more accurate:

"Some browsers pause script execution during scroll (e.g. some versions of iOS). If this option is selected the inview library will determine if elements became visible after the scroll events have stopped, once the javascript begins executing again."

Reply:
That's really helpful, thanks. How about this:
> "This setting relates to browsers that pause script execution whilst the learner is scrolling (e.g. Safari for iOS 6 & 7). This causes problems for components that use 'inview' completion. 'Scroll over' works around this by checking to see if content was made visible once scrolling has stopped. Unless you need support for browsers that pause script execution on scroll, you can disable this setting."

Reply:
Yup that's pretty close. It reads imprecisely in my brain with the order:  "... was made visible once.. [event occurrence].".

Could you rearrange the sentence such that event occurs and scroll over works around it afterwards?

"Once scrolling has stopped 'scroll over' works around the paused execution by checking if the content was made visible during the scroll."